### PR TITLE
feat: graphql-ws working

### PR DESCRIPTION
**Pull request by LeAlencar**

### CODE REVIEW

File: src/modules/transaction/subscriptions/newTransactionSubscription.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  transactionId: string
+}
+
+const TransactionNewSubscription = subscriptionWithClientId({
+  name: "TransactionNew",
+  inputFields: {},
+  outputFields: {
+    transaction: {
+      type: TransactionType,
+      resolve: async ({id}: any) => TransactionModel.findOne({_id: id})
```
Comment (vitorsouzaalmeida) [inline src/modules/transaction/subscriptions/newTransactionSubscription.ts:17]: Why are your type `any`?

Comment (LeAlencar) [reply src/modules/transaction/subscriptions/newTransactionSubscription.ts:17]: Actually I just forgot this one. Fixed on this commit here: https://github.com/LeAlencar/relay-graphql-api/commit/473557cd4a99934e85a2de19cb07d5715ce4c387

Comment (vitorsouzaalmeida) [reply src/modules/transaction/subscriptions/newTransactionSubscription.ts:17]: Nice one

File: src/modules/transaction/mutations/TransactionCreateMutation.ts [hunk 2]
```diff
@@ -27,17 +28,23 @@ const mutation = mutationWithClientMutationId({
...
     const transaction = await new TransactionModel({transactionId, name, category, price}).save();
 
+
     if (!transaction) {
       return {
         error: 'Transaction not found',
       };
     }
 
+    await pubSub.publish("NEW_TRANSACTION", { transactionId: transaction._id})
```
Comment (daniloab) [inline src/modules/transaction/mutations/TransactionCreateMutation.ts:38]: move `NEW_TRANSACTION` to a const

`const SUBSCRIPTIONS = { NEW_TRANSACTION: 'NEW_TRANSACTION'}`

File: src/schema/schema.ts [hunk 3]
```diff
@@ -2,8 +2,10 @@ import { GraphQLSchema } from 'graphql';
 
 import QueryType from './QueryType';
 import MutationType from './MutationType';
+import TransactionNewSubscription from './SubscriptionType';
```
Comment (daniloab) [inline src/schema/schema.ts:5]: Why not SubscriptionType?

**Pull request merged**